src/m0_facilitator/Container.ts

```diff
@@ -48,11 +51,14 @@ export default class Container {
     const configAuxChainId = config.facilitator.auxChainId;
     const subscriptions = await Subscriptions.create(
       transactionHandler,
-      repositories,
+      repositories.contractEntityRepository,
+      fetchQueries,
       config.facilitator.chains[configOriginChain].subGraphWs,
       config.facilitator.chains[configOriginChain].subGraphRpc,
+      subscriptionQueries.origin,
       config.facilitator.chains[configAuxChainId].subGraphWs,
       config.facilitator.chains[configAuxChainId].subGraphRpc,
+      subscriptionQueries.auxiliary,
     );
 
     const services = Services.create(repositories, config);
```

Comment:
We will also need Services class implementation for m1.

Reply:
I do not believe this should be part of this ticket. I think there is (should be) a separate ticket for Container class implementation.
/cc @deepesh-kn

Reply:
Resolving this as Services implementation is done